src/main/java/frc/robot/Robot.java

```diff
@@ -700,15 +768,15 @@ public void disabledInit() {
     visionLight.set(Relay.Value.kOff);
 
     Config.getInstance().reload();
+
+    mOperatorInterface.resetOverride();
```

Comment:
Wouldn't we also want to reset the override in `autonomousInit()`?

Reply:
No, because the robot has to be disabled before autonomous can be enabled.